README.rst

```diff
@@ -1,6 +1,18 @@
 PennyLane-Lightning-GPU Plugin
 ##############################
 
+.. header-start-inclusion-marker-do-not-remove
+
+❗ Warning
```

Comment:
what would you think of adding this warning to the installation page too?

I guess there should be some automated way to include this warning as a static header of all doc pages.

Reply:
We've decided this is too much, and will just remove the old docs and link to the new site instead. I'll undo this.